tracing-subscriber/src/registry/mod.rs

```diff
@@ -94,13 +94,20 @@ pub trait LookupSpan<'a> {
 
     /// Returns the [`SpanData`] for a given `Id`, if it exists.
     ///
-    /// **Note**: users of the `LookupSpan` trait should typically call the
-    /// [`span`] method rather than this method. The `span` method is
-    /// implemented by calling `span_data`, but returns a reference which is
+    /// <div class="information">
```

Comment:
sploded

![Screenshot from 2020-06-29 12-08-49](https://user-images.githubusercontent.com/1993852/86046000-4f83e900-ba01-11ea-94a2-66e4814fccf8.png)

Reply:
Gonna stop on this one, These all seem to be the same issue, I'm assuming once you fix it the rest will be minor formatting issues and it will be easier to review them without the sploded formatting everywhere distracting me.